LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2019, Kevin "KTOWN" Townsend for Adafruit Industries 
```

Comment:
Please update this to 2015 and "Limor Fried & Kevin Townsend for Adafruit Industries" to match the original commit.

Reply:
Done! :)